components/connector/internal/tokens/service.go

```diff
@@ -24,6 +25,8 @@ func NewTokenService(store Cache, generator TokenGenerator) *tokenService {
 }
 
 func (svc *tokenService) CreateToken(clientId string, tokenType TokenType) (string, apperrors.AppError) {
+	log.Infof("Generating token for %s with id %s", tokenType, clientId)
```

Comment:
there is a very similar log before entering this method.. 	

r.log.Infof("Generating one-time token for Application with id %s", appID)